Tests/Shared.Specs/EventAssertionSpecs.cs

```diff
@@ -744,7 +744,7 @@ public void When_monitoring_an_object_through_an_interface_it_should_monitor_onl
             }
         }
 
-#if NET45 || NET47
+#if NET45 || NET47 // TODO Why this is not available in e.g. core2?
```

Comment:
If you add `NETCOREAPP2_0` it does not compile, as `DefineDynamicAssembly` and `DefineDynamicModule` are not defined for that target.